zorka-core/src/main/java/com/jitlogic/zorka/core/integ/netuitive/JvmSystemStatsReport.java

```diff
@@ -12,9 +12,11 @@
 
     /* Logger */
     private final ZorkaLog log = ZorkaLogger.getLog(JvmSystemStatsReport.class);
+    private Java6CpuUsage java6CpuUsage;
```

Comment:
I meant in the entire project like https://github.com/Netuitive/zorka/blob/master/zorka-common/src/main/java/com/jitlogic/zorka/common/util/JSONWriter.java, https://github.com/Netuitive/zorka/blob/master/zorka-common/src/main/java/com/jitlogic/zorka/common/util/StringMatcher.java, and https://github.com/Netuitive/zorka/blob/master/zorka-core/src/main/java/com/jitlogic/zorka/core/ZorkaLib.java for example. I'd imagine it could be moved back into the constructor if it ever needed an argument.